README_v1.md

```diff
@@ -0,0 +1,129 @@
+# Stackable Operator for Apache Kafka
+
+[![Build Actions Status](https://ci.stackable.tech/job/kafka%2doperator%2dit%2dnightly/badge/icon?subject=Integration%20Tests)](https://ci.stackable.tech/job/kafka%2doperator%2dit%2dnightly)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://GitHub.com/stackabletech/kafka-operator/graphs/commit-activity)
+
+This is a Kubernetes Operator to manage [Apache Kafka](https://kafka.apache.org/) clusters.
+
+Looking to dive in? It's easy to get started with the Stackable data platform using `stackablectl`:
+
+```
+stackablectl spin up a demo
+```
+
+Read more in [link to fancy demo using stackablectl](#).
+
+## Installation
+
+To get the best experience, we recommended way to get started with Stackable is through the `stackablectl` tool.
+
+Read [the instructions below](#) to get started.
```

Comment:
Where would this link to? I think we can link to the docs from here. I think I'd like lots of links to the docs in general, as that _should_ be the main hub of information

Reply:
I'd link to the section in the README right below the line. It's a tiny thing, but personally I appreciate being able to click the text I want to go to.